packages/contracts/contracts/optimistic-ethereum/libraries/constants/Lib_PredeployAddresses.sol

```diff
@@ -8,16 +8,11 @@ library Lib_PredeployAddresses {
     address internal constant L2_TO_L1_MESSAGE_PASSER = 0x4200000000000000000000000000000000000000;
     address internal constant L1_MESSAGE_SENDER = 0x4200000000000000000000000000000000000001;
     address internal constant DEPLOYER_WHITELIST = 0x4200000000000000000000000000000000000002;
-    address internal constant ECDSA_CONTRACT_ACCOUNT = 0x4200000000000000000000000000000000000003;
-    address internal constant SEQUENCER_ENTRYPOINT = 0x4200000000000000000000000000000000000005;
     address payable internal constant OVM_ETH = 0x4200000000000000000000000000000000000006;
     // solhint-disable-next-line max-line-length
     address internal constant L2_CROSS_DOMAIN_MESSENGER = 0x4200000000000000000000000000000000000007;
     address internal constant LIB_ADDRESS_MANAGER = 0x4200000000000000000000000000000000000008;
     address internal constant PROXY_EOA = 0x4200000000000000000000000000000000000009;
-    // solhint-disable-next-line max-line-length
-    address internal constant EXECUTION_MANAGER_WRAPPER = 0x420000000000000000000000000000000000000B;
-    address internal constant SEQUENCER_FEE_WALLET = 0x4200000000000000000000000000000000000011;
```

Comment:
Are you sure that we want to delete the sequencer fee wallet? What would your proposed new method of handling fees be?

Reply:
Nevermind, I see that its at the bottom now